From 9596d0ed8627fe51c744a7a3a5fc59db4044dbef Mon Sep 17 00:00:00 2001
From: Matthew McCullough <matthew@github.com>
Date: Fri, 21 Sep 2012 14:11:24 -0700
Subject: [PATCH 1/2] Commit one

---
 test.rb | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 test.rb

diff --git a/test.rb b/test.rb
new file mode 100644
index 0000000..f4dbe7c
--- /dev/null
+++ b/test.rb
@@ -0,0 +1,6 @@
+require 'test/unit'
+
+class TestSomething < Test::Unit::TestCase
+  def test_assert
+  end
+end
\ No newline at end of file
-- 
1.7.12

